bom/runtime/src/main/resources/extensions-overrides.json

```diff
@@ -47,6 +42,16 @@
         ]
       }
     },
+    {
+      "name": "Core",
+      "id": "core",
+      "description": "Essential Quarkus components. Provided automatically",
```

Comment:
@emmanuelbernard is it still/always true that they are provided automatically (ex `logging-json` and `logging-sentry`)?

Reply:
Ah good point, something like 

"Basic Quarkus components: engine, log, etc." is probably a better description. Can you push it @ia3andy ?